crates/red_knot_python_semantic/resources/mdtest/annotations/string.md

```diff
@@ -173,3 +173,27 @@ p: "call()"
 r: "[1, 2]"
 s: "(1, 2)"
 ```
+
+## Multi line annotation
+
+Quoted type annotations should be parsed as if surrounded by parentheses.
+
+```py
+def f(
+    a1: """Literal[None]""",
+    a2: """(Literal[None])""",
+    a3: """(
+      int |
+      str
+  )
+  """,
+    a4: """
+     int |
+       str
+  """,
+):
+    reveal_type(a1)  # revealed: None
+    reveal_type(a2)  # revealed: None
+    reveal_type(a3)  # revealed: int | str
+    reveal_type(a4)  # revealed: int | str
```

Comment:
I assume out of these four, only `a4` failed before this PR? (Can't check right now, since GH git ops are down.)

Not sure what `a1` and `a2` are really testing?

Reply:
Yes only a4 fails. I added the other ones just to have a single line triple quoted in the tests but I guess it's not really needed. Will remove it.